Cannot find module './BrowserStackReporter'

[v1.1.0](https://github.com/karma-runner/karma-browserstack-launcher/releases/tag/v1.1.0) has an incorrect path in it which prevents any Browserstack tests from running.

```
ERROR [plugin]: Error during loading "myrepo/node_modules/karma-browserstack-launcher" plugin:
  Cannot find module './BrowserStackReporter'
```

The line responsible for this is `index.js`, specifically

```
var BrowserStackReporter = require('./BrowserStackReporter')
```

which should be

```
var BrowserStackReporter = require('./browserstack-reporter')
```

--- a/index.js
+++ b/index.js
@@ -3,7 +3,7 @@ var api = require('browserstack')
 var BrowserStackTunnel = require('browserstacktunnel-wrapper')
 var os = require('os')
 var workerManager = require('./worker-manager')
-var BrowserStackReporter = require('./BrowserStackReporter')
+var BrowserStackReporter = require('./browserstack-reporter')
 
 var createBrowserStackTunnel = function (logger, config, emitter) {
   var log = logger.create('launcher.browserstack')

